src/panels/config/integrations/ha-config-integrations.ts

```diff
@@ -255,9 +261,81 @@ class HaConfigIntegrations extends SubscribeMixin(LitElement) {
     }
   );
 
+  public connectedCallback() {
+    this._configEntries = undefined;
+    super.connectedCallback();
+    if (this.hasUpdated) {
+      this._subscribeConfigEntries();
+    }
+  }
+
+  public disconnectedCallback() {
+    super.disconnectedCallback();
+    this._unsubscribeConfigEntries();
+  }
+
+  private _subscribeConfigEntries() {
+    this._subscribedConfigEntries = subscribeConfigEntries(
+      this.hass,
+      (messages) => {
+        let fullUpdate = false;
+        const newEntries: ConfigEntryExtended[] = [];
+        messages.forEach((message) => {
+          if (message.type === null || message.type === "added") {
```

Comment:
So the first response will be all integrations with type = `null`?

Reply:
Yes. Current state is a non event so null

Reply:
added a comment in 6a5e3d94ae42bc2f40921bf316d08d8c8fc98467